proxyclient/m1n1/hv.py

```diff
@@ -677,7 +683,7 @@ def rh(base, off, width):
             print(f"R {base:x}+{off:x}:{width} = 0x{data:x} -> 0x{ret:x}")
             return ret
 
-        for addr in (0x23b700420, 0x23d280098, 0x23d280088, 0x23d280090):
+        for addr in (0x23b700420, 0x23d280098, 0x23d280088):
```

Comment:
Does this cover the case where the HV is using both ports? I've only tested with port 0 so far, I imagine the problem addresses are different for each.

Reply:
I haven't noticed a difference after removing `0x23d280090` and still removing both ports from the guest's ADT.

It is however required when running running Mac OS as guest using HV on port 1. In that case it's necessary to hook `(0x23b700448, 0x23d2800a0, 0x23d280090)` though. I'll add that to this PR.